models/movie.js

```diff
@@ -83,4 +84,12 @@ const movieSchema = mongoose.Schema({
     },
 });
 
+movieSchema.pre('save', async function(next) {
```

Comment:
After researching a bit online, it seems `pre('save'` runs AFTER validation.
I can make this model save correctly by using the `pre('validate'` hook instead.